packages/ckeditor5-block-quote/docs/features/block-quote.md

```diff
@@ -17,8 +17,9 @@ Use the editor below to see the block quote plugin in action.
 
 Here are some other CKEditor 5 features that you can use similarly to the block quote plugin to structure your text better:
 
-* The {@link features/indent block indentation feature} allows you to set indentation for text blocks such as paragraphs or lists.
-* The {@link features/code-blocks code block feature} allows for insertion of various code listings.
+* {@link features/indent Block indentation feature}  &ndash; Set indentation for text blocks such as paragraphs or lists.
```

Comment:
In the guides above we got rid of "feature" (block indentation feature etc.), maybe we can also get rid of them in other guides?